packages/terra-dev-site/tests/jest/webpack/loaderUtils/configHelpers.test.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+
+const configHelpers = require('../../../../src/webpack/loaderUtils/configHelpers');
+
+describe('getNamespace', () => {
+  it('returns the namespace when the directory does not include packages or node_modules', () => {
+    const directory = '/directory/';
+    const namespace = 'namespace';
+    const result = configHelpers.getNamespace(directory, namespace);
+    expect(result).toEqual(namespace);
+  });
+
+  it('returns the package name immediately following the node_module directory', () => {
+    const directory = '/node_modules/directory/';
+    const namespace = 'namespace';
+    const result = configHelpers.getNamespace(directory, namespace);
+    expect(result).toEqual('directory');
+  });
+
+  it('returns the scoped package name immediately following the node_module directory', () => {
+    const directory = '/node_modules/@cerner/directory/';
+    const namespace = 'namespace';
+    const result = configHelpers.getNamespace(directory, namespace);
+    expect(result).toEqual('@cerner/directory');
+  });
+
+  it('returns the package name from the package.json associated with the mono repo package.', () => {
+    const directory = path.resolve(__dirname, 'testContent', 'packages', 'module', 'file');
```

Comment:
several paths in theses tests needed to be updated based on the mono repo dir structure.